tools/certify/commands/auth-ca.cpp

```diff
@@ -0,0 +1,127 @@
+#include "../keyio.hpp"
```

Comment:
This looks clumsy. Use INCLUDE_DIRECTORIES property in CMake if necessary to get rid of relative paths here.

Reply:
I'll have a look, I'm still new to CMake.